cypher/cypher-docs/src/docs/dev/administration.asciidoc

```diff
@@ -3,25 +3,9 @@
 
 [abstract]
 --
-This chapter explains how to use Cypher to administer Neo4j databases, such as creating databases, managing indexes and constraints, and managing security.
+This chapter explains how to use Cypher to administer Neo4j databases, for the purpose of managing indexes, and managing security.
 --
 
-Neo4j supports the management of multiple databases within the same DBMS.
-The metadata for these databases, including the associated security model, is maintained in a special database called the `system` database.
-Most administrative commands must be executed against the `system` database because they involve editing the metadata for the entire system.
-This includes all commands related to managing multiple databases as well as all commands for defining the security model: users, roles and privileges.
-The administrative commands that are specific to the schema of an individual database are still executed against that specific database.
```

Comment:
A lot of this is still valid... really only the part about managing databases are not.

```
Neo4j supports the management of multiple databases within the same DBMS.
The metadata for these databases, including the associated security model, is maintained in a special database called the `system` database.
Most administrative commands must be executed against the `system` database because they involve editing the metadata for the entire system.
This includes all commands related to defining  and managing the security model: users, roles and privileges.
The administrative commands that are specific to the schema of an individual database are still executed against that specific database.
```

However it should probably be stated in the new database chapter that the commands should be run against the system database...

Reply:
yep, I thought it was covered reasonably in the introduction to the Databases chapter:
<img width="865" alt="Screenshot 2021-07-23 at 09 25 35" src="https://user-images.githubusercontent.com/12032446/126756105-3d653dc5-e678-4462-b214-4c57acbc836d.png">

What do you think? I could incorporate some of the wording in this removed section?

As the Administration chapter itself is due to be removed, I thought it wasn't worth replacing this text with anything specific to indexes or security.

Reply:
I think the databases are covered then yes.

I only rewrote 
`This includes all commands related to managing multiple databases as well as all commands for defining the security model: users, roles and privileges.` 
to 
`This includes all commands related to defining  and managing the security model: users, roles and privileges.`
the rest of the intro for admin section remained the same

Reply:
ah cool - I'll return that rewritten intro then (at least until we move the indexes & security!)

Sorry, I had missed that revision in your comment above!